low-level/files/opt/seagate/sspl/setup/generate_sspl_bundle/sspl_bundle_generate

```diff
@@ -25,12 +25,15 @@ import json
 import subprocess
 # Add the top level directories
 sys.path.insert(0, f'/opt/seagate/cortx/sspl/low-level')
-from framework.platforms.realstor.realstor_enclosure import singleton_realstorencl
```

Comment:
As discussed  with @satish-seagate, I moved this import statement to where it is really required.
Keeping it here unnecessarily retries enclosure login when `-noencl` argument is passed.